### DISCUSSION TITLE
Problems with speed control for Corsair H100i XT Pro.

### DISCUSSION THREAD

**Discussion by burrisd** [EvanMulawski/FanControl.CorsairLink#192] [Q&A]
Referring to Fan Control version 206 (NET 8.0) with CorsairLink 1.7.0 Beta 2, I can see the pump and fans and I see both pump and fan speeds that seem reasonable. However, in trying to take manual control form calibration, I am seeing indeterminate behavior trying to control the speed of fan 1 & 2 and pump. At times, it appears to work with very slow adaptation to the selected speed. Other times, even within the same session, the actual speeds indicated appear to have no correlation to the speed control setting. Also, intermittently the speeds appear to change fast or maximum with no input from the controls. I assume this is somehow related to the firmware of the device believing it must adjust the speed, even though I see no temperature indication that would confirm that.

Bottom line, the speed control reliably affecting the fan/pump speeds, appears intermittent (or not working at all).

I am NOT running IQUE, so there could potentially be a older firmware running on the cooler.

Can anyone confirm reliable control of the H100i XT Pro using the Fan Control speed controls?
Is there a minimum firmware version requirement requiring a firmware update to get this working?

My guess is that the firmware is either ignoring the speed inputs, or it is actively fighting with its own attempts to control the speed.

Appreciate any help!
